lib/puma/request.rb

```diff
@@ -12,6 +12,13 @@ module Puma
   #
   module Request
 
+    LEN_BUFFER   = 1_024 * 256
+    STRM_LEN_MAX = 1_024 * 1_024 * 4
```

Comment:
What is "strm"?

Reply:
Short for stream.  Some of the code here calls the `IOBuffer` object `stream`, and some of the current code calls it `lines`.

Would it be better to call it `io_buffer`?

Also, much of the code uses `io` for the client socket.  Should it be called `socket`?

And, thanks again, when I'm writing code, I often use short names.  Need to stop doing that.

Reply:
I like your suggestions

Reply:
See the last commit, renamed as mentioned above.  Also, I often needed to remind myself what `res_info` was.  Changed it to `resp_info`.  I see `res`, and I don't immediately think of `response`...